trtools/utils/tr_harmonizer.py

```diff
@@ -1582,6 +1611,17 @@ def HasQualityScore(self) -> bool:
         # would only be reachable if VcfTypes is expanded in the future)
         _UnexpectedTypeError(self.vcftype)  # pragma: no cover
 
+    def IsBeagleVCF(self) -> bool:
+        """
+        Is this a VCF produced by running the Beagle software to impute STRs from a panel generated by an TR genotyper?
+
+        See Also
+        --------
+        tr_harmonizer.IsBeagleVCF
+        """
+        global IsBeagleVCF
```

Comment:
globals make me nervous. do we need to use a global variable here?

Reply:
I believe I was incorrectly using this. I thought this was necessary to access the file-scoped function defined outside the class. But this doesn't seem to be the case, so I'm just removing this statement.